tests/unit/test_parameters/test_parameter_sets/test_OKane2020.py

```diff
@@ -21,6 +21,7 @@ def test_load_params(self):
     def test_functions(self):
         root = pybamm.root_dir()
         param = pybamm.ParameterValues("Chen2020_plating")
+        param.update({"Lithium plating kinetic rate constant [m.s-1]": 1E-10})
```

Comment:
why is this done? why not update the test value?

Reply:
The two papers used different values for this parameter.